pandas/core/indexes/multi.py

```diff
@@ -1078,6 +1078,27 @@ def levshape(self) -> Shape:
 
     @property
     def codes(self) -> tuple:
+        """
+        Codes of the MultiIndex.
+
+        Codes are the integer labels for each level designating which label at each location.
```

Comment:
I think the wording is a little bit awkward here, maybe:
```suggestion
        Codes are the integer labels for each level designating the position of the index value in a sorted list of all index values in that level.
```
Maybe a bit too long though ;)